subscription_platform/dashboards/monitor_saasboard__active_subscriptions.dashboard.lookml

```diff
@@ -0,0 +1,950 @@
+    sorts: [daily_active_logical_subscriptions.date_date desc]
+    limit: 1
+    column_limit: 50
+    custom_color_enabled: true
+    show_single_value_title: true
+    show_comparison: false
+    comparison_type: value
+    comparison_reverse_colors: false
+    show_comparison_label: true
+    enable_conditional_formatting: false
+    conditional_formatting_include_totals: false
+    conditional_formatting_include_nulls: false
+    x_axis_gridlines: false
+    y_axis_gridlines: true
+    show_view_names: false
+    show_y_axis_labels: true
+    show_y_axis_ticks: true
+    y_axis_tick_density: default
+    y_axis_tick_density_custom: 5
+    show_x_axis_label: true
+    show_x_axis_ticks: true
+    y_axis_scale_mode: linear
+    x_axis_reversed: false
+    y_axis_reversed: false
+    plot_size_by_field: false
+    trellis: ''
+    stacking: ''
+    limit_displayed_rows: false
+    legend_position: center
+    point_style: none
+    show_value_labels: false
+    label_density: 25
+    x_axis_scale: auto
+    y_axis_combined: true
+    show_null_points: false
+    interpolation: linear
+    y_axes: [{label: '', orientation: left, series: [{axisId: daily_active_logical_subscriptions.logical_subscription_count,
+            id: daily_active_logical_subscriptions.logical_subscription_count, name: Logical
+              Subscription Count}], showLabels: true, showValues: true, unpinAxis: true,
+        tickDensity: default, tickDensityCustom: 5, type: linear}]
+    x_axis_zoom: true
+    y_axis_zoom: true
+    defaults_version: 1
+    listen:
+      Country: countries.name
+      Region: countries.region_name
+      Has Refunds (Yes / No): current_subscription_state.has_refunds
+      Has Fraudulent Charges (Yes / No): current_subscription_state.has_fraudulent_charges
+      Service ID: subscription_services.id
+      Payment Provider: daily_active_logical_subscriptions.subscription__payment_provider
+      Plan Interval: daily_active_logical_subscriptions.subscription__plan_interval
+      Plan: daily_active_logical_subscriptions.subscription__plan_summary
+      Active Date: daily_active_logical_subscriptions.date_date
+    row: 8
+    col: 0
+    width: 8
+    height: 8
+  - title: Daily Active Subscriptions
+    name: Daily Active Subscriptions
+    model: subscription_platform
+    explore: daily_active_logical_subscriptions
+    type: looker_line
+    fields: [daily_active_logical_subscriptions.date_date, daily_active_logical_subscriptions.logical_subscription_count]
+    filters:
+      daily_active_logical_subscriptions.was_active_at_day_end: 'Yes'
+    sorts: [daily_active_logical_subscriptions.date_date desc]
+    limit: 5000
+    column_limit: 50
+    x_axis_gridlines: false
+    y_axis_gridlines: true
+    show_view_names: false
+    show_y_axis_labels: false
+    show_y_axis_ticks: true
+    y_axis_tick_density: default
+    y_axis_tick_density_custom: 5
+    show_x_axis_label: false
```

Comment:
A bunch of the X axis labels were annoyingly auto-named like "Date Date" and "Month Month", but rather than tweak all those I instead hid most of them where it seemed like they were self-explanatory.  If you want any axis labels added back let me know.

Reply:
I like them removed as well, but good to know if we get feedback that they are preferred.